src/components/AddExploitation/index.jsx

```diff
@@ -0,0 +1,170 @@
+import { useState, useContext, useEffect } from 'react';
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import { useHistory } from 'react-router-dom';
+import User from '../../contexts/User';
+import StyledExploitation from './styles';
+
+function AddExploitation() {
+  const history = useHistory();
+  const { user } = useContext(User);
+  const idUser = user.id;
+  const [datas, setDatas] = useState();
```

Comment:
Always give an initial value, if only to be sure the variable has the right type right off the bat